src/toast/tests/ops_time_constant.py

```diff
@@ -0,0 +1,137 @@
+# Copyright (c) 2015-2020 by the parties listed in the AUTHORS file.
+# All rights reserved.  Use of this source code is governed by
+# a BSD-style license that can be found in the LICENSE file.
+
+import os
+
+import numpy as np
+import numpy.testing as nt
+
+from astropy import units as u
+from astropy.table import Column
+
+from .mpi import MPITestCase
+
+from ..noise import Noise
+
+from .. import ops as ops
+
+from ..vis import set_matplotlib_backend
+
+from ..pixels import PixelDistribution, PixelData
+
+from .. import qarray as qa
+
+from ._helpers import create_outdir, create_satellite_data, create_fake_sky, fake_flags
+
+
+XAXIS, YAXIS, ZAXIS = np.eye(3)
+
+
+class TimeConstantTest(MPITestCase):
+    def setUp(self):
+        fixture_name = os.path.splitext(os.path.basename(__file__))[0]
+        self.outdir = create_outdir(self.comm, fixture_name)
+
+    def test_time_constant(self):
+
+        # Create a fake satellite data set for testing
+        data = create_satellite_data(self.comm)
+
+        # Create an uncorrelated noise model from focalplane detector properties
+        default_model = ops.DefaultNoiseModel(noise_model="noise_model")
+        default_model.apply(data)
+
+        # Simulate noise from this model
+        sim_noise = ops.SimNoise(noise_model="noise_model", out="signal")
+        sim_noise.apply(data)
+
+        # Copy the signal for reference
+        for obs in data.obs:
+            obs.detdata.ensure("signal0")
+            for det in obs.local_detectors:
+                obs.detdata["signal0"][det][:] = obs.detdata["signal"][det]
```

Comment:
This is fine, but for reference another way would be:
```
copy = ops.Copy(detdata=[("signal", "signal0")])
copy.apply(data)
```

Reply:
That is much cleaner! I changed it.